cognite/neat/rules/exporters/_rules2dms.py

```diff
@@ -42,17 +56,19 @@ class DMSExporter(CDFExporter[DMSSchema]):
 
     def __init__(
         self,
-        export_components: frozenset[Literal["all", "spaces", "data_models", "views", "containers"]] = frozenset(
-            {"all"}
-        ),
+        export_components: Component | Collection[Component] = "all",
         include_space: set[str] | None = None,
```

Comment:
not sure what `include_space` means? isn't space already specified in `rules`?

Reply:
This is a leftover from the last DMSImporter. I guess something from AXO, maybe we should just remove the option.